src/assets/i18n/fi.json

```diff
@@ -0,0 +1,1274 @@
+{
+  "authentication": {
+    "title": "Tietosuojaa koskeva vaikutustenarviointi",
+    "subtitle1": "Alusta tietosuojaa koskevien <br>  vaikutustenarviointien luomiseen ja hallintaan",
+    "subtitle2": "Työkalut ja sanastot",
+    "start_app": "Tietosuojaa koskevan vaikutustenarvioinnin ohjelmisto (beeta)",
+    "description": "Tämä <strong>Ranskan tietosuojaviranomaisen (CNIL)</strong> laatima ohjelmisto on tarkoitettu avustamaan rekisterinpitäjiä yleisen tietosuoja-asetuksen vaatimusten täyttämisessä ja sen osoittamisessa. Ohjelmisto mahdollistaa tietosuojaa koskevan vaikutustenarvioinnin tekemisen CNIL:n kehittämällä menetelmällä.",
+    "start": "Aloita"
+  },
+  "header": {
+    "logo_title": "Tietosuojaa koskeva vaikutustenarviointi",
+    "logo_subtitle": "",
+    "homepage_link": "Ohjauspaneeli",
+    "knowledge_base_link": "Tietopankki",
+    "pia_example_link": "Esimerkki vaikutustenarvioinnista",
+    "languages": {
+      "official_translations": "Viralliset käännökset",
+      "cnil_translations": "CNIL:n tarjoamat käännökset",
+      "unofficial_translations": "Epäviralliset käännökset",
```

Comment:
This is "community_translations" in i18n/en.json 
Also, it's good to see you, Finland.